build/build.generic.cmd

```diff
@@ -0,0 +1,38 @@
+@echo off
+
+rem
+rem Parameters
+rem   %1: vs_solution_version: VS solution version: VS2012, VS2013, VS2015
+rem   %2: vs_configuration:    VS configuration:    Debug, Release
+rem   %3: vs_target:           target:              Build or Rebuild
+rem   %4: vs_platform:         platform:            x64 or Win32
+rem   %5: vs_toolset:          toolset:             v100, v110, v120, v140
+rem
+
+set vs_solution_version=%1
+set vs_configuration=%2
+set vs_target=%3
+set vs_platform=%4
+set vs_toolset=%5
+
+set msbuild="%windir%\Microsoft.NET\Framework\v4.0.30319\MSBuild.exe"
+IF %vs_solution_version% == VS2013 SET msbuild="C:\Program Files (x86)\MSBuild\12.0\Bin\MSBuild.exe"
+IF %vs_solution_version% == VS2015 SET msbuild="C:\Program Files (x86)\MSBuild\14.0\Bin\MSBuild.exe"
```

Comment:
I *think* that before VS2013, msbuild was part of the .NET framework, and thus installed under the .NET Framework build folder. Starting from VS2013, they shipped it as part of VS , under its own folder (I remember TeamCity was having problems with VS2013 initially, because of that change, and I had invoke msbuild myself for a while)

See https://blogs.msdn.microsoft.com/visualstudio/2013/07/24/msbuild-is-now-part-of-visual-studio/